src/web_service.iced

```diff
@@ -328,6 +328,38 @@ class HackerNewsBinding extends SocialNetworkBinding
 
 #==========================================================================
 
+class GenericSocialBinding extends SocialNetworkBinding
+  constructor : (args) ->
+    @remote_service = args.remote_service
+    if (r = args.name_regexp)? then try @name_regexp = new RegExp(r)
+    super args
+
+  @single_occupancy : () -> false
+  single_occupancy  : () -> GenericSocialBinding.single_occupancy()
```

Comment:
These will be sort of single_occupancy. Each user will have a slot for each service. Just like right now each user has one slot for each Twitter proof. Maybe returning `false` here is good even if not the whole picture, I'm not sure where this function is used.